src/mlpack/core/data/string_encoding_policies/bag_of_words_encoding_policy.hpp

```diff
@@ -0,0 +1,152 @@
+/**
+ * @file bag_of_words_encoding_policy.hpp
+ * @author Jeffin Sam
```

Comment:
I think it would be fair to list @lozhnikov as an author as well, applies to the other files as well.